src/main/java/com/gk/portfolio/entities/Feedback.java

```diff
@@ -0,0 +1,44 @@
+package com.gk.portfolio.entities;
+
+import javax.persistence.Entity;
+
+
+@Entity
+public class Feedback extends IdentifiedEntity<Long>{
```

Comment:
Let's add a possibility to add Social account URLs of feedback owner (URL should be validated)